### PR TITLE
Use 'UK' and column comments

### PR DESCRIPTION
**Pull request by justintime**
Mermaid supports a key type of 'UK' for unique key, better to use that instead of adding a comment. Also, if the SQLAlchemy column has a comment assigned, we should add that to the Mermaid comment.

Here's the output for a table has has both a unique key and a comment in SQLAlchemy after the changes in this pull request:

```
  games {
    INTEGER id PK
    INTEGER away_team_id FK
    INTEGER home_team_id FK
    INTEGER league_id FK "nullable"
    INTEGER season_id FK
    INTEGER tracked_team_id FK
    INTEGER user_id FK
    INTEGER away_score
    VARCHAR(255) cuid UK "These are generated by clients and should be unique"
    DATE game_date
    INTEGER home_score
    VARCHAR(255) location
    BOOLEAN overtime
    BOOLEAN shootout
  }
```

### CODE REVIEW

**Comment by tedivm** [comment]
Can you fix the errors from testing? You should be able to pull down the branch and run `make chores` to do so.

**Comment by justintime** [comment]
> Can you fix the errors from testing? You should be able to pull down the branch and run `make chores` to do so.

Thanks for the `make` hint.  I was able to get `make chores` run cleanly on that last commit.  I also added a column comment on FK's that describes which table the column refers to.  On my ERD with ~ 20 tables and tons of FK's, it can be tedius to trace lines.

**Comment by justintime** [comment]
I can update the test suite as well, brb

File: paracelsus/transformers/mermaid.py [hunk 1]
```diff
@@ -29,15 +30,16 @@ def _column(self, column: Column) -> str:
                 column_str += " PK"
         elif len(column.foreign_keys) > 0:
             column_str += " FK"
+            options.append(f"Foreign key references {column.table.name}")
```
Comment (tedivm) [inline paracelsus/transformers/mermaid.py:33]: Can you make this a separate pull request? I'm not sure I want this comment in there, especially when we have the relationships drawn out already with the diagram itself.

File: paracelsus/transformers/mermaid.py [hunk 2]
```diff
@@ -29,15 +30,16 @@ def _column(self, column: Column) -> str:
...
+        elif column.unique:
+            column_str += " UK"
 
-        options = []
+        if column.comment:
+            options.append(f"{column.comment}")
```
Comment (tedivm) [inline paracelsus/transformers/mermaid.py:38]: Since column.comment is a string the f string and quotes are redundant-

In other words both of these lines are the same, so it should use the succinct one:

```python
options.append(column.comment)
options.append(f"{column.comment}")
```

File: tests/transformers/test_mermaid.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 def test_mermaid(metaclass):
     mermaid = Mermaid(metaclass=metaclass)
     graph_string = str(mermaid)
+    print(graph_string)
```
Comment (tedivm) [inline tests/transformers/test_mermaid.py:7]: You left a debugging statement here.

**Review by tedivm — CHANGES_REQUESTED** [review]
This looks good but I put a few comments in for changes. Thanks for putting this together!

**Review by tedivm — APPROVED** [review]
